charts/newrelic-logging/templates/persistentvolume.yaml

```diff
@@ -0,0 +1,52 @@
+{{- if not (empty .Values.fluentBit.persistence) }}
+{{- if and (eq .Values.fluentBit.persistence.mode "persistentVolume") (not .Values.fluentBit.persistence.persistentVolume.existingVolumeClaim) }}
+{{- if and (not .Values.fluentBit.persistence.persistentVolume.dynamicProvisioning) (not .Values.fluentBit.persistence.persistentVolume.existingVolume) }}
```

Comment:
I see 3 ifs here and just 2 closing "ends" at the end of the file, is this correct? 🤔 

Also, can these conditions be probably simplified as in just a single "if":
```
{{- if and (eq .Values.fluentBit.persistence.mode "persistentVolume") 
  (not .Values.fluentBit.persistence.persistentVolume.existingVolumeClaim) 
  (not .Values.fluentBit.persistence.persistentVolume.dynamicProvisioning) 
  (not .Values.fluentBit.persistence.persistentVolume.existingVolume) }}
```

(don't know if multiline ifs are allowed though)

Reply:
I tried and it failed so I assume helm only allow 2 conditions but will recheck.